contrib/gitian-descriptors/gitian-linux-parallel.yml

```diff
@@ -0,0 +1,141 @@
+---
+name: "zcash-5.3.0-rc1"
+enable_cache: true
+distro: "debian"
+suites:
+- "buster"
+- "bullseye"
+architectures:
+- "amd64"
+packages:
+- "curl"
+- "autoconf"
+- "automake"
+- "bsdmainutils"
+- "binutils-gold"
+- "ca-certificates"
+- "faketime"
+- "g++-multilib"
+- "git-core"
+- "libc6-dev"
+- "libtinfo5"
+- "libtool"
+- "libxml2"
+- "m4"
+- "ncurses-dev"
+- "pkg-config"
+- "python"
+- "unzip"
+- "wget"
+- "zlib1g-dev"
+remotes:
+- "url": "https://github.com/zcash/zcash.git"
+  "dir": "zcash"
+files: []
+script: |
+  WRAP_DIR=$HOME/wrapped
+  HOSTS="x86_64-linux-gnu"
+  CONFIGFLAGS="--enable-glibc-back-compat --enable-reduce-exports --disable-bench --enable-hardening --enable-werror"
+  MAKEOPTS="V=1 -j$(nproc)"
+  FAKETIME_HOST_PROGS=""
+  FAKETIME_PROGS="date ar ranlib nm strip objcopy"
+  HOST_CFLAGS=""
+  HOST_CXXFLAGS=""
+  HOST_LDFLAGS=-static-libstdc++
+
+  export QT_RCC_TEST=0
+  export GZIP="-9n"
+  export TAR_OPTIONS="--mtime="$REFERENCE_DATE\\\ $REFERENCE_TIME""
+  export TZ="UTC"
+  export BUILD_DIR=`pwd`
+  mkdir -p ${WRAP_DIR}
+  if test -n "$GBUILD_CACHE_ENABLED"; then
+    export SOURCES_PATH=${GBUILD_COMMON_CACHE}
+    export BASE_CACHE=${GBUILD_PACKAGE_CACHE}
+    mkdir -p ${BASE_CACHE} ${SOURCES_PATH}
+  fi
+
+  function create_global_faketime_wrappers {
+  for prog in ${FAKETIME_PROGS}; do
+    echo '#!/usr/bin/env bash' > ${WRAP_DIR}/${prog}
+    echo "REAL=\`which -a ${prog} | grep -v ${WRAP_DIR}/${prog} | head -1\`" >> ${WRAP_DIR}/${prog}
+    echo 'export LD_PRELOAD=/usr/lib/x86_64-linux-gnu/faketime/libfaketime.so.1' >> ${WRAP_DIR}/${prog}
+    echo "export FAKETIME=\"$1\"" >> ${WRAP_DIR}/${prog}
+    echo "\$REAL \$@" >> $WRAP_DIR/${prog}
+    chmod +x ${WRAP_DIR}/${prog}
+  done
+  }
+
+  function create_per-host_faketime_wrappers {
+  for i in $HOSTS; do
+    for prog in ${FAKETIME_HOST_PROGS}; do
+        echo '#!/usr/bin/env bash' > ${WRAP_DIR}/${i}-${prog}
+        echo "REAL=\`which -a ${i}-${prog} | grep -v ${WRAP_DIR}/${i}-${prog} | head -1\`" >> ${WRAP_DIR}/${i}-${prog}
+        echo 'export LD_PRELOAD=/usr/lib/x86_64-linux-gnu/faketime/libfaketime.so.1' >> ${WRAP_DIR}/${i}-${prog}
+        echo "export FAKETIME=\"$1\"" >> ${WRAP_DIR}/${i}-${prog}
+        echo "\$REAL \$@" >> $WRAP_DIR/${i}-${prog}
+        chmod +x ${WRAP_DIR}/${i}-${prog}
+    done
+  done
+  }
+
+  export PATH=${WRAP_DIR}:${PATH}
+
+  # Faketime for depends so intermediate results are comparable
+  create_global_faketime_wrappers "2000-01-01 12:00:00"
+  create_per-host_faketime_wrappers "2000-01-01 12:00:00"
+
+  cd zcash
+  BASEPREFIX=`pwd`/depends
```

Comment:
There are changes to this file that aren't reflected in `gitian-linux.yml` (e.g., using `pwd` rather than `${PWD}`). Are these necessary for the parallelization, or are they independent changes that should be made in `gitian-linux.yml` as well?

Reply:
```suggestion
  BASEPREFIX="`pwd`/depends"
```